### ISSUE TITLE
DM/Channel search gone after app redesign?

### ISSUE THREAD

**Issue by bleomycin** [RocketChat/Rocket.Chat.iOS#2050]
- Your Rocket.Chat app version: 3.0.1
- Your Rocket.Chat server version: 0.66.3
- Device (or Simulator) you're running with: Iphone X

It seems that i'm no longer able to search the contents of DM's or Channels after the ios app update to 3.0? Perhaps I just can't figure out how to anymore? If not are there plans to bring this feature back? Thanks for the amazing software, it's very much appreciated!

**Comment by rafaelks** [comment]
@bleomycin Yes, it'll be added again in the next releases, was a fail in our side. Thank you!
